examples/datasource-basic/playwright.config.ts

```diff
@@ -0,0 +1,52 @@
+import type { PluginOptions } from '@grafana/plugin-e2e';
+import { defineConfig, devices } from '@playwright/test';
+import { dirname } from 'node:path';
+
+const pluginE2eAuth = `${dirname(require.resolve('@grafana/plugin-e2e'))}/auth`;
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig<PluginOptions>({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: 'http://localhost:3000',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    // 1. Login to Grafana and store the cookie on disk for use in other tests.
+    {
+      name: 'auth',
+      testDir: pluginE2eAuth,
+      testMatch: [/.*\.js/],
+    },
+    // 2. Run tests in Google Chrome. Every test will start authenticated as admin user.
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'], storageState: 'playwright/.auth/admin.json' },
+      dependencies: ['auth'],
+    },
+  ],
+});
```

Comment:
Would it make sense to have this basic config under the `.config/` folder as well, and only extend that one in this file?

(Like this we could make it easier in the future to make changes to the base config, and users could also extend it without interfering with our future changes.)

Reply:
Hmm, yeah, that's probably a good point. (Something that we should probably discuss later is - should we provide a basic playwright config under .config/ for new scaffolds, or just leave playwright out from this pattern?)

Reply:
Even though not ideal to have a different pattern for Playwright, I think it's worth making an exception here and leave the playwright conf in the root. If not, it can become pretty messy if for example we'd like the `update` cmd to make changes to the playwright config. 

Any thoughts @jackw?

Reply:
Yeah, I am not against making an exception in this case. (How do we imagine the `update` cmd updating the playwright config in case it sits in the plugin root directory?)

Reply:
I'm not entirely sure tbh. Actually I'm struggling to see when we would ever want to update the Playwright config. The plugin author should have complete control over the config.